docs/src/lib/internals/formats.md

```diff
@@ -1,5 +0,0 @@
-# Formats
```

Comment:
The `Format` module can be completely removed in a follow up PR, I got some complaints about missing docs here so removed the doc-page for now.